src/pages/Login/index.tsx

```
@@ -3,123 +3,250 @@ import { useForm } from "react-hook-form"
+    const onSubmit = async (data: LoginFormValues) => {
+        try {
+            setLoginError("")
+            const res = await login(data.email, data.password)
+            console.log("Login success:", res)
+            if (res.access_token) {
+                localStorage.setItem("token", res.access_token)
+                navigate("/")
+            } else {
+                setLoginError("An unexpected error occurred")
+            }
+        } catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)